README.md

```diff
@@ -72,7 +95,7 @@ encode.cancel!
 encode.cancelled?  # true
 ```
 
-An encoding job is meant to be the record of the work of the encoding engine and not the current state of the outputs.  Therefore moved or deleted outputs will not be reflected in the encoding job.
+An encode object job is meant to be the record of the work of the encoding engine and not the current state of the outputs.  Therefore moved or deleted outputs will not be reflected in the encoding job.
```

Comment:
I like this change to using `encode object` here.  What do you think about removing `job` and making the change to `encode object` in the next sentence as well?
```suggestion
An encode object is meant to be the record of the work of the encoding engine and not the current state of the outputs.  Therefore moved or deleted outputs will not be reflected in the encode object.
```

Reply:
@cjcolvar That's fine with me -- note that "encoding job" came from the existing README, that's terminology you were already using (although not always consistently). I can change it _everywhere_   including in lines I hadn't previously touched, to consistently say "encode object"?  Everywhere in README anyway. 

I'll add another commit doing that universally?

Reply:
@cjcolvar actually... that's not so straightforward. Can you look through the whole existing README for `job`, and tell me what you think we should do about terminology/usage? Maybe you do want me to leave it alone everywhere else, and just change this sentance?

I'm not sure. "Encoding job" and "encode object" are both used in README, and I think sometimes they are synonyms, and other times "job" refers to think thing actually running on AWS and "object" refers to the `ActiveEncode::Base` model representing it?

Reply:
I don't want to hold up this PR so I think we can revisit the terminology used in a separate issue.   I think changing this one sentence is good for now.